core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitConfig.java

```diff
@@ -30,6 +30,9 @@ public class ConsensusCommitConfig {
   public static final String ASYNC_COMMIT_ENABLED = PREFIX + "async_commit.enabled";
   public static final String ASYNC_ROLLBACK_ENABLED = PREFIX + "async_rollback.enabled";
 
+  public static final String PARALLEL_FILL_READ_SET_ENABLED =
+      PREFIX + "parallel_fill_read_set.enabled";
```

Comment:
This is a difficult one...
```suggestion
      PREFIX + "parallel_read_for_blind_mutation.enabled";
```

Reply:
Fixed in ed8cc7ac7d2684b9a70c8d5e28be9225de8a8ac5. It's renamed to `scalar.db.consensus_commit.parallel_implicit_pre_read.enabled`.